### ISSUE TITLE
Add the ability to play when Clementine is stopped

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tjofle/clementine-webremote#4]
```
You can't start clementine if it stopped. This patch adds the ability to do so 
(I didn't add an icon because I don't know where the originals came from; feel 
free to add an icon if you want.)

```

Original issue reported on code.google.com by `ty...@tycho.ws` on 23 Oct 2011 at 4:12

Attachments:
- [play_when_stopped.patch](https://storage.googleapis.com/google-code-attachments/clementine-webremote/issue-4/comment-0/play_when_stopped.patch)

**Comment by GoogleCodeExporter** [comment]
```
This has now been implemented in v0.3
```

Original comment by `tjo...@gmail.com` on 27 Aug 2012 at 5:58

**Comment by GoogleCodeExporter** [comment]
Original comment by `tjo...@gmail.com` on 18 Oct 2012 at 7:11
- Changed state: **Fixed**
